packages/builder/src/builder.ts

```diff
@@ -385,19 +385,31 @@ export async function getOutputs(
 // TODO: this func is dumb but I need to walk through this time period before I want to turn it into something of beauty
 function addOutputsToContext(ctx: ChainBuilderContext, outputs: ChainArtifacts) {
   const imports = outputs.imports;
-
   for (const imp in imports) {
     ctx.imports[imp] = imports[imp];
+    //also add simplified syntax
+    ctx[imp] = { ...imports[imp], contracts: undefined };
+    delete ctx[imp].contracts;
```

Comment:
does this remove the "complicated" syntax? for backwards compatibility we want to keep this syntax alive for the time being.